src/App.js

```diff
@@ -1,25 +1,21 @@
-import logo from './logo.svg';
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import './App.css';
+import Home from './components/Home';
+import Charts from './components/Charts';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    return (
+        <div className="App">
+            <Router>
+                <main>
+                    <Switch>
+                        <Route path="/" exact component={Home} />
+                        <Route path="/charts" component={Charts} />
+                    </Switch>
+                </main>
+            </Router>
+        </div>
+    );
```

Comment:
Why do you need the `main` to wrap the Switch?

Reply:
removed main